src/app/views/query-runner/request/Request.tsx

```diff
@@ -114,6 +135,21 @@ export class Request extends Component<IRequestComponent, any> {
     );
   }
 
+  private handlePivotItemClick = (pivotItem?: PivotItem) => {
+    if (!pivotItem) {
+      return;
+    }
+    this.onPivotItemClick(pivotItem);
+    this.toggleFeedback(pivotItem);
+  }
+
+  private toggleFeedback = (event: any) => {
+    const { key } = event;
```

Comment:
Adding this could help make the tab fully behave like a button without losing focus on other opened tabs.

```suggestion
    event.preventDefault();
    const { key } = event;
```